error.go

```diff
@@ -27,12 +27,12 @@ import (
 )
 
 type (
-	// ErrorKind is an enum listing the possible types of errors.
+	// ErrorId is an enum listing the possible types of errors.
 	ErrorKind int
 
```

Comment:
The above looks like a search and replace was a bit too eager :) Maybe the type should also be renamed to `ErrorId`?